components/istio-installer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20190325-ff66a3a
+FROM eu.gcr.io/sap-se-cx-kyma-wookiee/alpine-kubectl:kubernetes-1.16
 
```

Comment:
I think we should use official images, created and hosted in kyma-project gcr repo

Reply:
Yes, I will prepare PRs, I've needed those to set up test cluster.